# Acceleration
mission_1:
  alpha_smooth: 3.0

  safe_speed: 5.0
  speed_limit: 10.0
  EBS_speed_limit: 12.0

  acceleration_speed_limit: 7.0
  accel_min: 0.0
  accel_max: 2.0
  lateral_acceleration_limit: 14.0

  path_similar_mse: 0.1
  safe_lap: true

# Skidpad
mission_2:
  alpha_smooth: 3.0

  safe_speed: 5.0
  speed_limit: 10.0
  EBS_speed_limit: 12.0

  acceleration_speed_limit: 7.0
  accel_min: -2.0
  accel_max: 2.0
  lateral_acceleration_limit: 14.0

  path_similar_mse: 0.1
  safe_lap: true

# Trackdrive
mission_3:
  alpha_smooth: 3.0

  safe_speed: 5.0
  speed_limit: 10.0
  EBS_speed_limit: 12.0

  acceleration_speed_limit: 7.0
  accel_min: -2.0
  accel_max: 2.0
  lateral_acceleration_limit: 14.0

  path_similar_mse: 0.1
  safe_lap: true
  
# Uses Emergency Brakes System (EBS) speed limit
mission_4:
  # Smoothening parameter, used as weight in cost function for penalizing large accelerations
  alpha_smooth: 3

  # Speed in lap 1 or if optimization fails
  safe_speed: 5
  EBS_speed_limit: 12
  acceleration_speed_limit: 14
  speed_limit: 10
  lateral_acceleration_limit: 4
  accel_min: -2
  accel_max: 2
  safe_lap: true

  # If the MSE of positions and previous path positions is below,
  # speed profile computation will be skipped and previous solution is used.
  path_similar_mse: 0.1